engine/index.js

```diff
@@ -0,0 +1,52 @@
+import alexa from './lib/alexa.js';
+import redisHelper from './lib/redis-helper.jd';
+
+function setCurrentSnapshot(redis, snapshotKey) {
```

Comment:
The term 'snapshot' implies to me that all the data is from a particular instant in time. Our crawls are likely to take days, if not weeks, to complete. Maybe let's prefer the term 'crawl' over 'snapshot'?